src/tests/shamalgs/primitives/scan_exclusive_sum_in_placeTests.cpp

```
@@ -74,17 +74,18 @@ NEW_TEST(Unittest, "shamalgs/primitives/scan_exclusive_sum_in_place", 1) {
         }
     };
 
+    if (!shamalgs::primitives::impl::is_impl_set_scan_exclusive_sum_in_place()) {
+        shamalgs::primitives::impl::autoselect_impl_scan_exclusive_sum_in_place();
+    }
     auto current_impl = shamalgs::primitives::impl::get_current_impl_scan_exclusive_sum_in_place();
```

Cppcheck (2.21.0)

[style] The function 'assert_equal_array' is never used.

(unusedFunction)